### ISSUE TITLE
Browserosaurus apparently no longer launching with other login items since updating to v20.6.1

### ISSUE THREAD

**Issue by sdc1-clickclack** [will-stone/browserosaurus#670]
### Is there an existing issue for this?

- [X] I have searched the existing issues
- [X] I have searched the [discussions forum](https://github.com/will-stone/browserosaurus/discussions)
- [X] I understand this form is for reporting bugs, and not for requesting [support for new browsers or apps](https://github.com/will-stone/browserosaurus/blob/main/guide/supporting-a-browser-or-app.md).

### Current Behaviour

Hi, Will. I am the sole user of my Mac. I have both a regular user account and an admin account, and I always have Browserosaurus among the automatic login items for both accounts. While logged into my admin account today, I learned that you had released v20.6.1, so I updated to that new version. I also updated some security software (Intego) that required me to restart my Mac, so I dutifully complied. When the restart was complete and I logged into my admin account, I immediately saw an unfamiliar dialog box saying that Browserosaurus was an app downloaded from the Internet and asking me to confirm that I wanted to open it. Naturally, I clicked the Yes option — but Browserosaurus did not appear in my menu bar, so I launched it manually from my Applications folder and it appeared in the menu bar. After completing admin-related tasks, I logged out of the admin account and logged into my regular user account. Once again, I noticed that Browserosaurus did not appear in my menu bar despite still being listed among the apps I have designated to launch automatically at login.

### Expected Behaviour

If Browserosaurus is one of the apps specified for automatic launch at login, the expected behavior is that it will launch by the time all other items have appeared on the far right-hand side of the menu bar. In my case, the app did not launch automatically when I logged in to either my admin or regular user accounts even though I verified that it is still listed as one of the apps to be launched at login.

### Steps To Reproduce

1. After updating to Browserosaurus v20.6.1, log out and log in again (or restart your Mac ).
2. Look for dialogs requesting confirmation before launching Browserosaurus, and respond in the affirmative.
3. Wait for login processes to complete, at which time icons for all apps automatically launched at login should appear in the menu bar.
4. Determine whether the Browserosaurus icon is present in the menu bar, and also whether it is indeed running or not.

### Browserosaurus version

20.6.1

### macOS version

12.7.1

### CPU Architecture

ARM

### Anything else?

Nothing to add, other than that I currently do not recall this happening before.

**Comment by will-stone** [comment]
Hi. Strange, as this seems like an OS-level issue. Could you please restart again and see if B loads? Thanks.

**Comment by sdc1-clickclack** [comment]
Thanks for your reply, Will. I just restarted again, logged into my regular user account, and after waiting for all login processes and auto-launched apps to complete, there was no **B** in my menu bar. I also checked the Activity Monitor and did not see **B** listed. I looked for it alphabetically first, and then used the search feature to be thorough, but it was not there. I chose my regular user account for this test because, just to be sure, I had deleted and then re-added **B** among my login items for this account.
If no-one else is reporting this, I wonder if it is related to my still running MacOS Monterey (12.7.1). Not that it is likely to be relevant, but my Mac is a 16-inch MacBook Pro with M1 Max.

**Comment by sdc1-clickclack** [comment]
Whoa, okay, hold everything. I logged into my admin account and **B** now **did** appear in the menu bar. I returned to my regular user account, once again deleted and re-added **B** to my automatic login items, then logged out. When I logged back in to my regular user account, sure enough, now **B** launched automatically as it has always done before.

I'm stumped. (Maybe gremlins?) 

At any rate, it seems as though somehow the issue has been resolved. I'm confused, but no longer concerned. Thanks for your help, regardless of why this transient issue appeared.
